ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -128,6 +135,46 @@ message GetFlatTransactionResponse {
   Transaction transaction = 1;
 }
 
+message GetContractByIdRequest {
+  // The contract id being queried
+  // Required
+  string contract_id = 1;
+}
+message GetContractByIdResponse {
+  // The collection of events, will only contain consumed ExerciseEvents
+  repeated TreeEvent events = 1;
```

Comment:
Don't we want an offset here for the lat one?

Reply:
In the initial version the ById returns at most 2x rows so we don't need a `last` for the time being.